src/box/lua/schema.lua

```diff
@@ -16,6 +16,11 @@ local check_param = utils.check_param
 local check_param_table = utils.check_param_table
 local update_param_table = utils.update_param_table
 
+local DEFAULT_ORIGIN = ''
+-- We use the field ID instead of the field name so we don't need to upgrade
+-- the schema to use _origins option.
+local PRIV_OPTS_FIELD_ID = 6
+
```

Comment:
We use a strict schema for system tables so I think we do need upgrade. The upgrade function should set the format for the new field and add an empty map to each existing privilege. Please don't forget to implement and test downgrade as well.

Reply:
Ok, I will do in in the separate PR.

Reply:
I think that the new logic shouldn't depend on the upgrade to simplify things for a user.

However, the upgrade is a good thing to assign a name for the 6th `_priv` field for better introspection.

Reply:
For now I suggest to leave upgrade for another PR, if we decide to execute it.